features/ALB003_Alignment-directions.feature

```diff
@@ -1,11 +1,8 @@
 @implementer-agreement
 @ALB
 Feature: ALB003 - Alignment Nesting
```

Comment:
File name disagrees.

Reply:
Thanks for noticing. It's fixed, also added a test for it checks this automatically 
https://github.com/buildingSMART/ifc-gherkin-rules/commit/52d2a6f2aac9f7e5f669e39edf31165273695eb7